bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -170,11 +170,11 @@ private boolean updateTitle() {
 
             var issue = project.issue(id);
             if (issue.isPresent()) {
-                var issueTitle = issue.get().title();
+                var issueTitle = issue.get().title().strip();
```

Comment:
The title() method is called in a lot of places, perhaps it would be better to perform the stripping closer to the source?

Reply:
I'm not sure what you mean by that? We did use `issue.get().title()` in lots of places, but I replaces them with `issueTitle `to only have a single strip. That's as close to the source I can see possible.

Reply:
I was thinking of usages outside of this particular function, perhaps like https://github.com/openjdk/skara/blob/dbef34b8c9785ec185a70365324063a02b1ec010/bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java#L256 - we probably don't want trailing tabs or spaces there either.. So instead of searching for more of these, my suggestion was to add a call to strip before returning from the title() method to avoid similar problems.

Reply:
Seems a bit more risky since this can have effects on lots of other code. But sure, I fix it. I did try to look through the existing calls to title() and at least make a quick check that it seemed okay to strip the title.

Reply:
Thanks! Yeah if this change breaks anything, I think it's better that we find out now rather than worry about it. :)